modules/change-stream/change-stream.js

```diff
@@ -5,6 +5,7 @@
 
 angular.module('ls.ChangeStream',[]).factory('createChangeStream', ['$rootScope', function($rootScope) {
   function createChangeStream(eventSource, $scope) {
+    var stream = require('stream')
```

Comment:
This is an issue with the file `change-stream.js` because it's not being prepared for being loaded in the browser, as opposed to `stream.js` in the same folder (the file that's being included with the `require` statement. The latter file seems to be browserified which makes it that it works in browsers.

I've tried running the `browserify` command on `change-stream.js` and that makes it work, but it also makes the file huge and adds a lot of cruft.

That's why it would make sense to have this done in a build-step, also before running the tests in PhantomJS.